libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -3368,13 +3377,32 @@ export interface IDashboardCommand {
     readonly type: DashboardCommandType;
 }
 
+// @internal (undocumented)
+export interface IDashboardContentBaseProps {
+    backend?: IAnalyticalBackend;
+    config?: DashboardConfig;
+    dashboard?: string | ObjRef | IDashboard;
+    filterContextRef?: ObjRef;
+    permissions?: IWorkspacePermissions;
+    workspace?: string;
+}
+
+// @alpha (undocumented)
+export interface IDashboardContentCustomizer {
+    withCustomDecorator(providerFactory: (next: DashboardContentComponentProvider) => OptionalDashboardContentComponentProvider): IDashboardContentCustomizer;
+}
+
 // @public
 export interface IDashboardCustomComponentProps {
     // @alpha
     ButtonBarComponent?: CustomButtonBarComponent;
     // @alpha
     DashboardAttributeFilterComponentProvider?: OptionalAttributeFilterComponentProvider;
     // @alpha
+    DashboardContentComponentProvider?: OptionalDashboardContentComponentProvider;
+    // @alpha
+    DashboardContentProvider?: OptionalDashboardContentComponentProvider;
```

Comment:
Looks like the provider is duplicated (we should likely keep only DashboardContentComponentProvider)

Reply:
Removed duplicated DashboardContentProvider.